setup.py

```diff
@@ -334,6 +334,11 @@ def make_release_tree(self, base_dir, files):
             os.environ.get("CIBUILDWHEEL", "0") != "1"
         ) else []
     ),
+    extras_require={
```

Comment:
I think this is not doing what you would expect it to.

Python version specific markers can be achieved with

```
importlib-resources>=3.2.0; python_version<"3.10"
```

as per https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies.

This should be in the main dependencies section, not as an extra.

Reply:
This does actually work, I checked when I had the same thought previously. It is a bit outdated, and what you linked is the more recommended way, but this is valid (if perhaps not fully future-proof, as it is not well documented any more).